src/pages/home/index.jsx

```diff
@@ -1,41 +1,140 @@
           <Box sx={{ marginBottom: 5 }} key={props.title}>
             <QuestionCard showFilterResume editableTitle {...props} />
           </Box>
         ))}
       </Stack>
-    </Box>
+      </Box>
+      <Box
+        padding={{ xs: "20px", sm: "50px" }}
+        sx={{
+          backgroundColor: theme.palette.secondary.main,
+        }}
+      >
+        <Stack
+          spacing={4}
+          flexWrap="wrap"
+          direction={{ xs: "column", sm: "column", md: "row" }}
+          alignItems="center"
+          justifyContent="center"
+        >
+          <Typography
+            variant="h5"
+            sx={{
+              textAlign: "center",
+              lineHeight: "1.75",
+            }}
+          >
+            Want to make your contribution count? Create an account or sign in
+            on Open Food Facts!
+          </Typography>
+          <Stack direction="row">
+            <Button
+              variant="contained"
+              href="https://world.openfoodfacts.org/cgi/login.pl"
+              sx={{
+                backgroundColor: "white",
+                color: "black",
+                margin: "0 20px",
+              }}
+            >
+              Login
+            </Button>
+            <Button
+              variant="contained"
+              href="https://world.openfoodfacts.org/cgi/user.pl"
+              sx={{
+                backgroundColor: "white",
+                color: "black",
+              }}
+            >
+              Sign up
+            </Button>
+          </Stack>
+        </Stack>
+      </Box>
+      <Box textAlign="center">
+        <Button
+          onClick={handleOpen}
+          sx={{
+            margin: "20px auto",
+          }}
+        >
+          Learn why your contribution matters
+        </Button>
+      </Box>
+      <Modal
+        open={open}
+        onClose={handleClose}
+        aria-labelledby="modal-modal-title"
+        aria-describedby="modal-modal-description"
+      >
+        <Box
+        width={{xs:"90%",sm:"50%"}}
+          sx={{
+            backgroundColor: theme.palette.secondary.main,
+            color: theme.palette.secondary.contrastText,
+            position: "absolute",
+            left: "50%",
+            top: "50%",
+            transform: "translate(-50%, -50%)",
+            padding: "20px",
+          }}
+        >
+          <Typography id="modal-modal-title" variant="h6" component="h2" fontWeight={700}>
+            Your contribution matters
+          </Typography>
+          <Typography id="modal-modal-description" sx={{ mt: 2 }}>
+            Hunger Games is an effort to leverage the results of our OCR. When
+            you use Hunger Games by Open Food Facts, you improve our entire
+            database which helps the community make better food choices. We
+            gathers your annotations and use them to make Open Food Facts more
+            accurate and effecient. <br />
+            <br />
```

Comment:
better use `<p>` to divide text blocks as those are explicit on the start and end